app/services/idv/doc_auth_form_response.rb

```diff
@@ -1,10 +1,19 @@
 module Idv
   # A custom form response with additional noop methods to allow merging with DocAuth responses
   class DocAuthFormResponse < ::FormResponse
+    attr_accessor :ocr_pii
+
     def exception; end
 
     def pii_from_doc
       {}
     end
```

Comment:
what's the reason we differentiate between `pii_from_doc` and `ocr_pii`? I feel like kind of mean the same thing...but we've kind of turned `pii_from_doc` into our working space/sandbox for PII during proofing?